src/Backend/Modules/FormBuilder/Ajax/SaveField.php

```diff
@@ -247,6 +252,10 @@ public function execute(): void
             $settings['classname'] = \SpoonFilter::htmlspecialchars($classname);
         }
 
+        if ($autocomplete != '') {
```

Comment:
you can simplify this to `$settings['autocomplete'] = in_array($autocomplete, Autocomplete::POSSIBLE_VALUES) ? $autocomplete : '';`

This also removes the need for lines 61-63.